### PR TITLE
Chore: Add routes and controllers

### LINKED ISSUE #51: Create Initial Route Namespacing

### PR DESCRIPTION
**Pull request by MatthewTLim**
# Describe the changes below:
This PR will add all routes and controllers needed for set up.
# Relevant user story:
N/A
# Specific feedback requests? Include file name and line:
N/A
# Before submitting, check the following:
- [X] Entire test suite is passing
- [100%] SimpleCov test percentage 

# Note any co-authors / anything else you'd like to add:
Co-authored-by: Weston <[wtschutt@gmail.com](mailto:wtschutt@gmail.com)>
Co-authored-by: Dani <[danirae.powell@gmail.com](mailto:danirae.powell@gmail.com)>
Co-authored-by: Matt <[matlim200@gmail.com](mailto:matlim200@gmail.com)>
Co-authored-by: Mike <[mikewwood@icloud.com](mailto:mikewwood@icloud.com)>
Co-authored-by: Anna <[awiley33@gmail.com](mailto:awiley33@gmail.com)>”

### CODE REVIEW

**Review by westonio — APPROVED** [review]
Thanks for driving to set all of this up!

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+    resources :invoices, only: [:index, :show]
+  end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :admin do
+    get "/", to: "dashboard#index"
+    resources :merchants, only: [:index, :show]
+    resources :invoices, only: [:index, :show]
+  end
```
Comment (awiley33) [inline config/routes.rb:12]: This all looks good! Follows ReST

**Review by awiley33 — APPROVED** [review]
Looks good! Thanks for driving.

**Review by dani-wilson — APPROVED** [review]
Looks great! Thanks for doing that.

**Pull request merged**